### DISCUSSION TITLE
What is the correct spelling for max query depth configuration? graphql.servlet.max-query-depth=10 or graphql.servlet.maxQueryDepth=10

### DISCUSSION THREAD

**Discussion by allomy** [graphql-java-kickstart/graphql-spring-boot#862] [Q&A] [answered]
Hi, I'm confused about the correct spelling for the max query depth configuration, I find example for both `max-query-depth` or `maxQueryDepth`, I don't know which one is the right one.

Question 1: Which is the correct spelling in application.properties file? (A) or (B)? 
(A)
```
graphql.servlet.max-query-depth=10
```
(B) 
``` 
graphql.servlet.maxQueryDepth=10
```

Question 2: Which is the correct spelling in application.yml file? (C) or (D)?
(C)
```
graphql:
  servlet:
    maxQueryDepth: 10
```
(D)
```
graphql:
  servlet:
    max-query-depth: 10
```

**Comment by oliemansm** [comment, marked as answer]
Both work, since that's how the Spring Boot property resolution mechanism works. The kebab case variant is the naming convention recommended by Spring however. So that's A and D in your examples.

**Reply by allomy** [reply]
Thank you for your answer!
